app/template/nav.php

```diff
@@ -3,7 +3,7 @@
     ->addItem("Événements", "/evenements", "fa-calendar")
     ->addItem("Les licenciés", "/licencies", "fa-users")
     ->addItem("Mon profil", "/mon-profil", "fa-gear")
-    ->addItem("Suggestions", "/feedback", "fa-lightbulb")
+    ->addItem("Suggestions", "/feedback/new", "fa-lightbulb")
```

Comment:
In french ? We do that for the rest of the routes

Reply:
Solved